package-lock.json

```diff
@@ -1,22 +1,37 @@
 {
   "name": "content",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
What are these changes?

Reply:
These changes were made as part of an automatic process while I was using Git Code Spaces in a separate app (VS Code). They got pushed along with some of the other changes I made, if they're not necessary I'm happy to revert this file

Reply:
Please revert.